config/routes.rb

```diff
@@ -19,6 +19,7 @@
       post 'assisted-digital-survey-feedback', to: "assisted_digital_feedback#create", format: false
       post 'email-survey-signup', to: 'email_survey_signup#create', format: false
       post 'email-survey-signup.js', to: 'email_survey_signup#create', defaults: { format: :js }
+      post 'business-finder', to: 'business_finder#create', defaults: { format: :js }
```

Comment:
As with comments in https://github.com/alphagov/support-api/pull/278#pullrequestreview-198001674 I'd suggest we don't make this work single-use nor should we expose the specifics of client applications to broader systems like this.
I think a more generic approach is needed, we may well get more use out of this work if we wished to provide a content improvement feedback component elsewhere, and there's not much additional effort to rename it generically even if we do scrap it once the finder has run its course.